src/transformers/models/auto/modeling_tf_auto.py

```diff
@@ -163,6 +163,7 @@
     TFTransfoXLLMHeadModel,
     TFTransfoXLModel,
 )
+from ..wav2vec2.modeling_tf_wav2vec2 import TFWav2Vec2Model
```

Comment:
@sgugger I moved `TFWav2Vec2Model` to auto modeling. This worked for me locally, but I wanted to verify that I did this correctly. Could you take a look at it, please?

Reply:
Yes, that's the way to go!